From: Matt Fleming <matt.fleming@intel.com>
Date: Thu, 15 Nov 2012 20:03:16 +0000
Subject: [PATCH] efivarfs: Make efivarfs_fill_super() static
Git-commit: e83af1f18c78c7b6aa720beecc927ecc8afd3647
Patch-mainline: v3.8-rc1
Reference: fate#314499, fate#313527
Target: sle11-sp3

sparse is complaining that efivarfs_fill_super() doesn't have a
prototype. Make it static to avoid the warning.

Cc: Xie ChanglongX <changlongx.xie@intel.com>
Cc: Matthew Garrett <mjg@redhat.com>
Cc: Jeremy Kerr <jeremy.kerr@canonical.com>
Signed-off-by: Matt Fleming <matt.fleming@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>

---
 drivers/firmware/efivars.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/firmware/efivars.c
+++ b/drivers/firmware/efivars.c
@@ -1004,7 +1004,7 @@ static int efivarfs_unlink(struct inode
 	return -EINVAL;
 };
 
-int efivarfs_fill_super(struct super_block *sb, void *data, int silent)
+static int efivarfs_fill_super(struct super_block *sb, void *data, int silent)
 {
 	struct inode *inode = NULL;
 	struct dentry *root;
